lego/apps/articles/migrations/0012_rename_author_article_authors.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.9 on 2023-02-28 16:34
+
```

Comment:
You should delete the two migrations you have here and create a new one. The reason is that we don't want unnecessary migrations in prod. You can just delete both of there, reset your db locally and run makemimgrations again to fix it :)